db/migrate_core_cards/20141208132416_partial_reference_type.rb

```diff
@@ -0,0 +1,9 @@
+# -*- encoding : utf-8 -*-
+
+class PartialReferenceType < Wagn::CoreMigration
+  def up
+    Card.all.each do |card|
+      card.update_references
```

Comment:
So, find_each just makes a simpler loop, one nest rather than two.  Maybe if you were doing an update_all, you would get the ids in groups and do an update_all on the query for that set of ids, but we (often) need to do something to each card so we want the objects (in groups).